uber/config.py

```diff
@@ -72,6 +72,72 @@ def PREREG_BADGE_TYPES(self):
         return types
 
     @property
+    def BADGE_DISPLAY_TYPES(self):
+        """
+        There are several contexts where we want to display different badge types to select:
+        - A new attendee registering for the event
+        - An attendee claiming a blank badge in an existing group
+        - An existing attendee editing their registration
+
+        This property uses the parameters in the URL to figure out the context and return the correct
+        badge types to display for an attendee.
+        """
+        with sa.Session() as session:
+            params = cherrypy.lib.httputil.parse_query_string(cherrypy.request.query_string)
```

Comment:
Yeah, putting it in a custom tag was my first approach, but that quickly started taking too long to implement for what's really a fairly minor feature.

Jinja sounds nice. :( If we can't port this whole thing over to Django eventually, maybe we'll just refactor that in.